sroka/api/s3_connection/s3_connection_api.py

```diff
@@ -74,3 +75,32 @@ def s3_download_data(s3_filename, prefix=False, output_file=None, sep=','):
         data.to_csv(output_file, sep=sep)
 
     return data
+
+
+def s3_upload_data(data, bucket, path, sep=','):
+    key_id = config.get_value('aws', 'aws_access_key_id')
+    access_key = config.get_value('aws', 'aws_secret_access_key')
+    session = boto3.Session(
+        aws_access_key_id=key_id,
+        aws_secret_access_key=access_key
+    )
+
+    csv_buffer = StringIO()
+
+    try:
+        if type(data) == pd.core.frame.DataFrame:
+            data.to_csv(csv_buffer, sep=sep)
+        elif type(data) == np.ndarray:
+            np.savetxt(csv_buffer, data, delimiter=sep)
+    except AttributeError:
+        print('Uploaded file must be pandas DataFrame or numpy array')
```

Comment:
As @bckatarzyna mentioned in line 105, here it would also be good to extend the error message a bit by mentioning what type user actually supplied.
```suggestion
        print('Uploaded file must be pandas DataFrame or numpy array, found {}'.format(type(data)))
```